cypress/integration/cookieBannerWS.js

```diff
@@ -1,26 +1,36 @@
 import { worldServiceCookieBannerTranslations } from '../support/bodyTestHelper';
 import describeForEuOnly from '../support/describeForEuOnly';
-import { describeForLocalOnly } from '../support/limitEnvRuns';
+import { describeForLocalAndTest } from '../support/limitEnvRuns';
 import services from '../../src/app/lib/config/services';
 
 Object.keys(services).forEach(index => {
   const serviceConfig = services[index];
   const service = index;
 
   // This should be unhacked when this file is consolidated with other cookie testing files.
-  if (service === 'news' || service === 'default') {
+  if (
```

Comment:
Why cant we use a key in the service config for this? This style of if X exception is something we should reeeeallly avoid doing IMO

Reply:
I don't mind either way, my change was just proving that it's easy to fix.

Reply:
I can stick in a `isWorldService` into the cypress/support/config/services.js file. Not sure about the services not yet on test though.

How do you feel @benjaminhobbs @dr3 about amending the cypress/support/config/envs.js file to list the locales that a particular environment has or does not have, in an array.

If that suits you both, shall I go for a list of ones that they **do not** have - that way it's an ever reducing list that can eventually be removed.

Reply:
e.g.
``` 
test: {
    baseUrl: 'https://www.test.bbc.com',
    dataUrl: 'https://www.test.bbc.com',
    assetUrl: 'https://news.test.files.bbci.co.uk/include/articles/public',
    assetOrigin: 'https://news.test.files.bbci.co.uk',
    atiAnalyticsWSBucket: '598343',
    atiUrl: 'https://logws1363.ati-host.net?',
    lacksLocale: ['serbian', 'telugu', 'ukchina', 'zhongwen'],
  },
```

Reply:
I'll be honest I think we should merge this as is and tackle the refactor when we merge the tests. Doing it bit by bit will take too long.

The tests use various sources - some seem to use use src/app/lib/config/services/... as the driver, others use cypress/support/config/services.js.

I think a more consistent approach will mean we can leverage helpers further down the line. The tests works as expected so best to get these merged in.